### PR TITLE
chore: update blunderbuss.yml

### PR DESCRIPTION
**Pull request by tmatsuo**
I'll make another change at the beginning of June

### CODE REVIEW

**Review by bcoe — APPROVED** [review]

**Review by chingor13 — APPROVED** [review]

**Comment by tmatsuo** [comment]
Thanks! Let's wait for approval from @sofisl

**Review by sofisl — APPROVED** [review]

**Pull request merged**
